src/components/Rubric.vue

```diff
@@ -1,40 +1,168 @@
+            </span>
+            <div class="instr-form-row__item">
+              <label for="num-field-3">Numerical Score:</label>
+              <br>
+              <input
+                id="num-field-3"
+                class="feedback-row--progress instr-form-row__item--input"
+                type="number"
+                min="1"
+                max="4"
+                step="0.5"
+                placeholder="Whole or half numbers only"
+              />
+              <br>
+              <label for="text-field-3">Comments:</label>
+              <br>
+              <textarea
+                id="text-field-3"
+                class="instr-form-row__item--input"
+                rows="1"
+                placeholder="How can this student improve?"
+                required
+              />
+            </div>
+          </div>
+
+          <div class="instr-form-row">
+            <span class="s-h2 instr-form-row__item">
+              {{ this.$store.state.form.data.attributes.rubric_template[3].rubric_category_name }}
+            </span>
+            <div class="instr-form-row__item">
+              <label for="num-field-4">Numerical Score:</label>
+              <br>
+              <input
+                id="num-field-4"
+                class="feedback-row--progress instr-form-row__item--input"
+                type="number"
+                min="1"
+                max="4"
+                step="0.5"
+                placeholder="Whole or half numbers only"
+              />
+              <br>
+              <label for="text-field-4">Comments:</label>
+              <br>
+              <textarea
+                id="text-field-4"
+                class="instr-form-row__item--input"
+                rows="1"
+                placeholder="How can this student improve?"
+                required
+              />
+            </div>
+          </div>
         </div>
       </div>
     </div>
-    <hr class="feedback-container__dividing-line">
-    <div class="overall-container">
-      <span class="s-h2 feedback-container__category--label overall">Overall</span>
-      <div class="progress-bar-overall" id="bar-average">
-        <div class="progress-bar-overall__score-wrapper">
-          <div class="progress-bar-overall__score-wrapper--number">
-            AVERAGE SCORE
-          </div>
-        </div>
+
+<!--    <hr class="feedback-container__dividing-line">-->
+
+    <div class="instr-form-row">
+          <span class="s-h2 instr-form-row__item--title">
+            Overall
+          </span>
+      <div class="instr-form-row__item">
+        <label for="num-field-5">Numerical Score:</label>
+        <br>
+        <input
+          id="num-field-5"
+          class="feedback-row--progress instr-form-row__item--input"
+          type="number"
+          min="1"
+          max="4"
+          step="0.5"
+          placeholder="Whole or half numbers only"
+        />
```

Comment:
I think actually we don't want to collect an overall score from the instructor, since that's a calculation the BE is doing and sending back to us

Reply:
Oh excellent point, we can fix that next iteration.